src/app/config/config.ts

```diff
@@ -14,6 +14,7 @@ import {
   MailConfig,
 } from '../../types'
 
+import { sgid } from './features/sgid.config'
```

Comment:
we don't need to export this from the central config right? it can just be imported directly from the SGID config file when needed

Reply:
This is done to be consistent with the other feature config

Reply:
no leh, none of the other feature config is exported both from the config file and from the main config. e.g. `spcp-myinfo.config` is not.